src/responder/nss/nsssrv.c

```diff
@@ -401,7 +402,8 @@ static int sssd_supplementary_group(struct nss_ctx *nss_ctx)
      * the SSSD is compiled --with-sssd-user=sssd but the default of the
      * user option is root (this is what RHEL does)
      */
-    ret = sss_user_by_name_or_uid(SSSD_USER,
+    ret = sss_user_by_name_or_uid(&nss_ctx->ops,
```

Comment:
Can't we simply use `sss_sssd_user_uid_and_gid()` instead?

Reply:
I haven't changed this to have the smallest changes in the patch.

Reply:
ok